pkg/scalers/aws_cloudwatch_test.go

```diff
@@ -15,7 +15,7 @@ var testAWSCloudwatchResolvedEnv = map[string]string{
 }
 
 var testAWSAuthentication = map[string]string{
-	"awsAccessKeyId":     testAWSCloudwatchAccessKeyID,
+	"awsAccessKeyID":     testAWSCloudwatchAccessKeyID,
```

Comment:
This is a breaking change I presume @ahmelsayed? 🤔

Reply:
so all the doc was using `awsAccessKeyID`. so following the doc will lead to the logic https://github.com/kedacore/keda/blob/c0d86d70098272efd4215aea323041e34e577c2c/pkg/scalers/aws_iam_authorization.go#L27.  we can either change all the doc to `awsAccessKeyId` or making this change.. thought?

Reply:
Happy new year @tomkerkhove @ahmelsayed ! I'd like to follow up if there is anything that needs to be changed. 

As mentioned above, currently in documents are all with `awsAccessKeyID` but our code is with `awsAccessKeyId`. I changed the code to be all `awsAccessKeyID` (or we can do another way around, changing the doc instead).

Any recommendation for this? Thanks!

Reply:
Yeah, it's unfortunate it's not consistent in the code either between the metadata from the scaledobject and the authparams from the TriggerAuthentication. 

I'd rather update the docs, than break someone, but also not fixing it might cause issues for some down the road. I guess the other option is to allow for both casings for this particular value...

Reply:
Thanks @ahmelsayed and @tomkerkhove :)

I've updated `authParams` in aws_iam_authorization to allow both `awsAccessKeyId` and `awsAccessKeyID`. so it won't break someone who is using `awsAccessKeyId`.

I also merged all the changes to be one commit. Thank you very much!

Reply:
Thank you very much @msfuko for taking care of that. 

One final comment, if you can run `go fmt ./pkg/scalers` just to make sure formatting of the new files matched the go fmt guide, it would be great!

Reply:
Hi @ahmelsayed,
already pushed the formatted code! thanks!